docs/tutorial/getting-started.tex

```diff
@@ -0,0 +1,308 @@
+\documentclass[a4paper]{article}
+% generated by Docutils <https://docutils.sourceforge.io/>
+\usepackage{cmap} % fix search and cut-and-paste in Acrobat
+\usepackage{ifthen}
+\usepackage[T1]{fontenc}
+\usepackage{alltt}
+\usepackage{color}
+\setcounter{secnumdepth}{0}
+
```

Comment:
I think this .tex document shouldn't be generated. As far as I know, we work only with .rst in the repo.